### PR TITLE
Update humanic-ai-cohort.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Comment by Brandoncharleskhoo1** [comment]
Screenshot 2024-05-11 at 10 37 01 AM

File: docs/data/destinations/humanic-ai-cohort.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ description: Sync cohorts from Amplitude to Humanic AI
 status: new
 ---
 
-[Humanic AI](https://www.humanic.ai/) is an AI Powered PLG CRM for the Modern Revenue Team. This integration lets you sync cohorts from Amplitude to Humanic AI. 
+[Humanic AI](https://www.humanic.ai/) is a Marketing Automation Platform that helps you unlock growth. This integration lets you sync cohorts from Amplitude to Humanic AI. 
 
 !!!Tip
 
-    This integration is maintained by Humanic AI. Contact the [Humanic AI support team](https://help.Humanic AI.com/hc/en-us) for support with this integration. 
+    This integration is maintained by Humanic AI. Email **care@humanic.ai** for support with this integration.
```
Comment (markzegarelli) [inline docs/data/destinations/humanic-ai-cohort.md:11]:
```suggestion
    This integration is maintained by Humanic AI. Email [care@humanic.ai](mailto:care@humanic.ai) for support with this integration. 
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Comment by Brandoncharleskhoo1** [comment]
HI @markzegarelli i already implemented your suggestion so not sure why merging is still blocked

**Review by markzegarelli — APPROVED** [review]

**Comment by markzegarelli** [comment]
@Brandoncharleskhoo1 My bad, should be good to go now.

**Comment by arjunsaksena1** [comment]
Hi Team, 

Two changes are required: 

1. The first line needs to say "Humanic is an Agentic Marketing Platform that helps you unlock growth by using AI agents"
2. Humanic is free to use so we need to delete the second line under Considerations "You need to have a paid Humanic.ai plan to enable this integration."

Thank you!

**Pull request merged**